packages/app/src/overview/components/overview-screens/pure-overview/pure-overview.component.tsx

```diff
@@ -251,6 +251,16 @@ export const PureOverview: FunctionComponent<PureOverviewProps> = ({
     updateAllIndexes()
   }
 
+  const shouldErrorSyncModalVisible = (): boolean => {
```

Comment:
Why it's a method? It can easily be a variable

Reply:
for clarity of conditions 🙃 😉